tools/helpers/lxc.py

```diff
@@ -447,6 +446,55 @@ def shell(args):
     subprocess.run(command)
     if state == "FROZEN":
         freeze(args)
+        
+def custom_shell(args):
+    state = status(args)
+    if state == "FROZEN":
+        unfreeze(args)
+    elif state != "RUNNING":
+        logging.error("WayDroid container is {}".format(state))
+        return
+    command = ["lxc-attach", "-P", tools.config.defaults["lxc"],
+               "-n", "waydroid", "--clear-env"]
+    command.extend(android_env_attach_options())
+    if args.uid!=None:
+        command.append("--uid="+str(args.uid))
+    if args.gid!=None:
+        command.append("--gid="+str(args.gid))
+    elif args.uid!=None:
+        command.append("--gid="+str(args.uid))
+    if args.nolsm or args.allcaps or args.nocgroup:
+        elevatedprivs = "--elevated-privileges="
+        addpipe = 0
+        if args.nolsm:
+            if (addpipe == 1):
+                elevatedprivs+="|LSM"
+            else:
+                elevatedprivs+="LSM"
```

Comment:
```suggestion
            if addpipe:
                elevatedprivs+="|"
            elevatedprivs+="LSM"
```

Also please use False/True instead of 0/1 for addpipe